google-map-search.html

```diff
@@ -180,7 +186,7 @@
       }
 
       // Update location. This will trigger a new search.
-      this.location = new google.maps.LatLng(this.latitude, this.longitude);
+      this._setLocation(new google.maps.LatLng(this.latitude, this.longitude));
```

Comment:
just FYI, almost every place in v3 that takes a `LatLng` now takes a `LatLngLiteral`, (and `LatLngLiteral` are now preferred, FWIW). Probably better to `this._setLocation({lat: this.latitude, lng: this.longitude});`